internal/exec/stages/files/files.go

```diff
@@ -179,3 +188,21 @@ RemainAfterExit=yes`
 	_, err = f.WriteString(strings.Join(s.toRelabel, "\000") + "\000")
 	return err
 }
+
+func (s stage) addRelabelFile() error {
+	s.Logger.Info("adding relabel file")
+	defer s.Logger.Info("finished adding relabel file")
```

Comment:
Minor: let's just include the `relabelFilePath` itself in those logs to be more explicit?